it/uk/gov/hmrc/tai/integration/TaxCodeLatestSpec.scala

```diff
@@ -28,7 +28,8 @@ class TaxCodeLatestSpec extends IntegrationSpec {
 
 
   val apiUrl = s"/tai/$nino/tax-account/$year/tax-code/latest"
-  def request = FakeRequest(GET, apiUrl).withHeaders("X-SESSION-ID" -> generateSessionId)
+  def request = FakeRequest(GET, apiUrl)
+    .withHeaders("X-SESSION-ID" -> generateSessionId, "AUTHORIZATION" -> "Bearer 11")
```

Comment:
Use HeaderNames.xSessionId and HeaderNames.authorisation

Reply:
Updated